profile/perf_profile.go

```diff
@@ -0,0 +1,228 @@
+package profile
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/manifoldco/promptui"
+)
+
+var (
+	envPath = "PATH=" + os.Getenv("PATH")
+)
+
+// PerfProfileRequestsPerSeondBench profiles requests per second benchmark
+func PerfProfileRequestsPerSeondBench() error {
```

Comment:
typo in the name

Reply:
perf_profile.go is deleted and its functionality has been included in perf.go